apps/style/netsim/style.scss

```diff
@@ -510,78 +556,73 @@ svg {
         margin: 0 1em 10px 1em;
       }
     }
+  }
 
-    .netsim_my_device_tab {
-      .netsim_encoding_selector {
-        div.hiddenControl {
-          display: none;
-        }
+  .netsim-encoding-selector {
+    div.hidden-control {
+      display: none;
+    }
 
-        input[type="checkbox"] {
-          vertical-align: baseline;
-          margin: 0;
-        }
+    input[type="checkbox"] {
+      vertical-align: baseline;
+      margin: 0;
+    }
 
-        label {
-          display: inline;
-          margin-left: 0.25em;
-        }
-      }
+    label {
+      display: inline;
+      margin-left: 0.25em;
     }
+  }
 
-    .netsim_router_tab {
-      .netsim_router_log {
-        max-height: 200px;
-        overflow-y: scroll;
+  .netsim-router-log {
+    max-height: 200px;
+    overflow-y: scroll;
 
-        table {
-          th, td {
-            font-size: 8pt;
-          }
-        }
+    table {
+      th, td {
+        font-size: 8pt;
       }
     }
+  }
 
-    .netsim_dns_tab {
-      .dns_mode_control {
-        input[type="radio"] {
-          vertical-align: middle;
-          margin: 0;
-        }
+  .dns-mode-control {
+    input[type="radio"] {
+      vertical-align: middle;
+      margin: 0;
+    }
 
-        label {
-          display: inline;
-          margin-left: 1em;
-        }
-      }
+    label {
+      display: inline;
+      margin-left: 1em;
+    }
+  }
 
-      .netsim_dns_table {
-        tr.localNode {
-          font-weight: bold;
-          background-color: lightcyan;
-        }
+  .netsim-dns-table {
+    tr.local-node {
+      font-weight: bold;
+      background-color: lightcyan;
+    }
 
-        tr.dnsNode {
-          color: white;
-          background-color: darkorange;
-        }
-      }
+    tr.dns-node {
+      color: white;
+      background-color: darkorange;
```

Comment:
Color `darkorange` should be written in hexadecimal form as `#ff8c00`<br>Color literals like `darkorange` should only be used in variable declarations; they should be referred to via variable everywhere else.